html/pfappserver/lib/pfappserver/I18N/en.po

```diff
@@ -10954,7 +10954,7 @@ msgstr "Redis Port"
 
 # pf::action (VIOLATION_ACTIONS)
 msgid "reevaluate_access_action"
-msgstr "Reevaluate Access Action"
+msgstr "Isolate device on violation"
```

Comment:
I don't know if we always "isolate" - we could also just switch the user to an other VLAN, which isn't necessarily the isolation/quarantine VLAN.

Reply:
How does "Modify device VLAN on violation" work?

Reply:
Might not always be a VLAN :-)  It can be an ACL, a controller role, etc.

How about "Reevaluate network access" ?

Reply:
"Modify device network access on violation"?

Reply:
or "Change network access on violation" ? I hesitate between Change or Update or Modify. I would skip "device" to make it shorter.